### PR TITLE
add uax_url_email tokenizer docs

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
add uax_url_email tokenizer docs

### Issues Resolved
Part of https://github.com/opensearch-project/documentation-website/issues/1483 addressed in this PR.

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas @varun-lodaya Please provide tech review approval to move this PR forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

File: _analyzers/tokenizers/uax-url-email.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Generated tokens
+
+Use the following request to examine the tokens generated using the created analyzer:
```
Comment (kolchfa-aws) [inline _analyzers/tokenizers/uax-url-email.md:48]:
```suggestion
Use the following request to examine the tokens generated using the analyzer:
```

File: _analyzers/tokenizers/uax-url-email.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: UAX URL email
+parent: Tokenizers
+nav_order: 150
+---
+
+# UAX URL email tokenizer
+
+In addition to regular text, the `uax_url_email` tokenizer is designed to handle URLs, email addresses, and domain names. It is based on the Unicode Text Segmentation algorithm ([UAX #29](https://www.unicode.org/reports/tr29/)), which allows it to  correctly tokenize complex text, including URLs and email addresses.
```
Comment (natebower) [inline _analyzers/tokenizers/uax-url-email.md:10]:
```suggestion
In addition to regular text, the `uax_url_email` tokenizer is designed to handle URLs, email addresses, and domain names. It is based on the Unicode Text Segmentation algorithm ([UAX #29](https://www.unicode.org/reports/tr29/)), which allows it to correctly tokenize complex text, including URLs and email addresses.
```

**Review by natebower — COMMENTED** [review]
@kolchfa-aws @AntonEliatra Editorial review. Thanks!

**Review by udabhas — APPROVED** [review]
Looks good to me!

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**